### PR TITLE
#32

### PR DESCRIPTION
**Pull request by c-merchant**
Added limits to input fields.

### CODE REVIEW

File: src/app/account-management/general-information-form/general-information-form.component.html [hunk 1]
```diff
@@ -43,7 +43,7 @@
 
         <div class="input-group mb-3 col">
             <label>Facility Area</label>
-            <input type="text" class="form-control" formControlName="size" (input)="saveChanges()" onfocus="this.select();">
+            <input type="text" class="form-control" formControlName="size" (input)="saveChanges()" onfocus="this.select();" minlength="1" maxlength="999999999999999">
```
Comment (cranfillkd) [inline src/app/account-management/general-information-form/general-information-form.component.html:46]: I think you could probably limit this to around 60. At the point its 999999999999999, you wouldn't need to put a max on it.

File: src/app/account-management/sustainability-questions-form/sustainability-questions-form.component.html [hunk 2]
```diff
@@ -7,7 +7,7 @@
             <label>Reduction Goal</label>
             <div class="input-group mb-3">
                 <input type="number" class="form-control" formControlName="energyReductionPercent"
-                    (input)="saveChanges();">
+                    (input)="saveChanges();" min="0" max="999999999999999">
```
Comment (cranfillkd) [inline src/app/account-management/sustainability-questions-form/sustainability-questions-form.component.html:10]: Same as above

File: src/app/utility/energy-consumption/utility-meter-data/edit-electricity-bill/edit-electricity-bill.component.html [hunk 3]
```diff
@@ -26,7 +26,7 @@
         </label>
         <div class="input-group mb-3">
           <input id="totalEnergyUse" class="form-control" type="number" formControlName="totalEnergyUse"
-            [class.warning]="meterDataForm.controls.totalEnergyUse.value === 0" onfocus="this.select();">
+            [class.warning]="meterDataForm.controls.totalEnergyUse.value === 0" onfocus="this.select();" min="0" max="999999999999999999">
```
Comment (cranfillkd) [inline src/app/utility/energy-consumption/utility-meter-data/edit-electricity-bill/edit-electricity-bill.component.html:29]: You can remove the max from these. Same for the ones below

File: src/app/utility/energy-consumption/utility-meter-data/edit-utility-bill/edit-utility-bill.component.html [hunk 4]
```diff
@@ -14,7 +14,8 @@
       <label for="totalVolume">Total Volume <span class="required">*</span> </label>
       <div class="input-group mb-3">
         <input id="totalVolume" class="form-control" type="number" formControlName="totalVolume"
-          (input)="calculateTotalEnergyUse()" [class.warning]="meterDataForm.controls.totalVolume.value === 0" onfocus="this.select();">
+          (input)="calculateTotalEnergyUse()" [class.warning]="meterDataForm.controls.totalVolume.value === 0" onfocus="this.select();" 
+          min="0" max ="999999999999999">
```
Comment (cranfillkd) [inline src/app/utility/energy-consumption/utility-meter-data/edit-utility-bill/edit-utility-bill.component.html:18]: You can remove the max from these. Same for the ones below

**Review by cranfillkd — CHANGES_REQUESTED** [review]
Great work Charlotte. The main thing is to just remove or adjust the ones with 9e100.

**Comment by c-merchant** [comment]
Got it. Thank you!

**Review by cranfillkd — APPROVED** [review]

**Pull request merged**